smorest_crud/view.py

```diff
@@ -183,8 +183,7 @@ def post(self, args=None):
         # create
         item = self.model(**args)
 
-        # access check should be done in subclass (for now)
-        # self._check_can_create(item, args=args)
+        self._check_can_create(item, args=args)
```

Comment:
by enabling this, does it mean that every model implementing `AccessControlUser` needs to implement this method now?

Reply:
yes. But if we decided not to make this check so important I can remove `NotImplementedError` form it and make it return True by default.

Reply:
I believe it will be the best solution

Reply:
yeah 👍 make the common case simple
needing to implement specialized logic for create is probably not a common case imo

Reply:
make sense, thanks!